### PR TITLE
Api600/switch

### PR DESCRIPTION
**Pull request by AnushaY1916**
### Description
API 600 support for Switches and Switch Types

### Issues Resolved
NA

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
- [ ] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: spec/integration/resource/api600/c7000/switch/update_spec.rb [hunk 1]
```diff
@@ -11,19 +11,19 @@
...
 
-klass = OneviewSDK::API500::C7000::Switch
+klass = OneviewSDK::API600::C7000::Switch
 RSpec.describe klass, integration: true, type: UPDATE do
-  let(:current_client) { $client_500 }
-  include_examples 'SwitchUpdateExample  API300', 'integration api500 context'
+  let(:current_client) { $client_600 }
+  include_examples 'SwitchUpdateExample  API300', 'integration api600 context'
 
   describe '#set_scope_uris' do
```
Comment (soodpr) [inline spec/integration/resource/api600/c7000/switch/update_spec.rb]: This test case is not required

File: lib/oneview-sdk/resource/api600/synergy/switch.rb [hunk 2]
```diff
@@ -9,13 +9,13 @@
 # CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
 
-require_relative '../../api300/synergy/switch'
+require_relative '../../api500/synergy/switch'
 
 module OneviewSDK
   module API600
     module Synergy
       # Switch resource implementation for API 500 Synergy
```
Comment (soodpr) [inline lib/oneview-sdk/resource/api600/synergy/switch.rb]: it should be 600

File: lib/oneview-sdk/resource/api600/c7000/switch.rb [hunk 3]
```diff
@@ -9,20 +9,13 @@
 # CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
 
-require_relative '../../api300/c7000/switch'
+require_relative '../../api500/c7000/switch'
 
 module OneviewSDK
   module API600
     module C7000
       # Switch resource implementation for API 500 C7000
```
Comment (soodpr) [inline lib/oneview-sdk/resource/api600/c7000/switch.rb]: comment needs to be rectified.its 600 API

File: examples/shared_samples/switch.rb [hunk 4]
```diff
@@ -110,7 +112,7 @@
 # When a scope uri is added to a switch resource, this resource is grouped into a resource(enclosure, server hardware, etc.) pool.
 # Once grouped, with the scope it's possible to restrict an operation or action.
 # For the switch resource, this feature is only available for C7000 and api version higher than or equal to 300.
```
Comment (madhav-bharadwaj) [inline examples/shared_samples/switch.rb]: Need to change this comment maybe.

Comment (AnushaY1916) [reply examples/shared_samples/switch.rb]: Added accordinly

File: lib/oneview-sdk/resource/api600/c7000/switch.rb (hunk 3 above)
Comment (AnushaY1916) [reply lib/oneview-sdk/resource/api600/c7000/switch.rb]: Changed the version

File: lib/oneview-sdk/resource/api600/synergy/switch.rb (hunk 2 above)
Comment (AnushaY1916) [reply lib/oneview-sdk/resource/api600/synergy/switch.rb]: Changed the version

File: examples/shared_samples/switch.rb [hunk 5]
```diff
@@ -109,8 +111,8 @@
 # A scope defines a collection of resources, which might be used for filtering or access control.
 # When a scope uri is added to a switch resource, this resource is grouped into a resource(enclosure, server hardware, etc.) pool.
 # Once grouped, with the scope it's possible to restrict an operation or action.
-# For the switch resource, this feature is only available for C7000 and api version higher than or equal to 300.
-if @client.api_version >= 300 && variant == 'C7000'
+# For the switch resource, this feature is only available for C7000 and api version equal to 300.
+if @client.api_version == 300 && variant == 'C7000'
```
Comment (madhav-bharadwaj) [inline examples/shared_samples/switch.rb]: Hey @AnushaY1916 , as discussed yesterday, can you check this condition? Please test it against 3.10 DCS to check if endpoint does not work with 500 also.

Also, this condition is not in line with the endpoint support file.

**Pull request merged**